ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/tools/ObjStoreBrowserTest.java

```diff
@@ -91,7 +92,8 @@ public void testObjectStoreBrowser() throws Exception {
 	@Test
 	public void basicOSBTest () throws Exception
 	{
-		ObjStoreBrowser osb = new ObjStoreBrowser("os");
+		arjPropertyManager.getObjectStoreEnvironmentBean().setObjectStoreDir("os");
```

Comment:
The envinomentBean property already exists. Since we don't want the StoreDir value to change it would be better to set it in the environment bean only.

Reply:
Hmm, something to think about is that the ObjectStoreEnvironmentBean:::setObjectStoreDir could be seen as slightly different use case to what the OSB would use it for in that possibly an OSB user wants to connect to a different object store than the one that the transaction manager in memory is configured to use. That being said, I don't think that use case should be particularly common if at all and so I am not -1 to relying on that field. That being said, we would want a release note for it so we need to make sure the release note for JBTM-3973 users of the added getInstance method to understand the impact and of the environment bean config on the OSB.

Reply:
Good point, I'll add a note on the JBTM

Reply:
Actually the release note for this should go for the new JBTM that track the deprecation and removal of the constructor accepting the logDir as parameter I think. Do you agree @tomjenkinson ?

Reply:
Thinking again to your message I believe we need a release note for both the JBTMs. For the JBTM this PR addresses I wrote "Deprecating the ObjStoreBrowser constructors in favor of the getInstance method."